code/swb_base/InventoryBase.cs

```diff
@@ -0,0 +1,71 @@
+using System;
```

Comment:
imo should be merged with ours? Or did you just wrapped the swb system so we can use multiple add-ons? Afaik, you're referencing them directly.

Reply:
Removed `InventoryBase.cs`, not needed.